AI/EmptyAI/StdInc.h

```diff
@@ -4,4 +4,6 @@
 
 // This header should be treated as a pre compiled header file(PCH) in the compiler building settings.
 
-// Here you can add specific libraries and macros which are specific to this project.
\ No newline at end of file
+// Here you can add specific libraries and macros which are specific to this project.
+
+VCMI_LIB_USING_NAMESPACE

```

Comment:
Why it's needed in stdinc? Should be same for other ai libs?

Reply:
It is actually there in all other ai libs, even in develop branch.
Apparently EmptyAI was not tested/included originally, so it ended up without this change